File: NetBeansProjects/WebApplication1/src/java/com/epic/canvascontrollibrary/CCLImageSliderProps.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.epic.canvascontrollibrary;

import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author Gilgamesh
 */
    public class CCLImageSliderProps
    {
        public String CanvasID;
        public String WindowID;
        public String X;
        public String Y;
        public String Width;
        public String Height;
        public List<Object> ImageURLs;
        public String Direction;
        public String StepIncrement;
        public String ClickFunction;
        public String HoldForTicks;
        public String CurrentImageIndex;
        public String Slide;
        public String HoldCountDown;
        public String TabStopIndex;

        CCLImageSliderProps()
        {
            ImageURLs = new ArrayList<Object>();
        }
    }

File: ASP.NET MVC Web Api New Solution/CanvasControlLibrary/CanvasControlLibrary/Models/CCLImageSliderProps.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace CanvasControlLibrary.Models
{
    public class CCLImageSliderProps
    {
        public string CanvasID { get; set; }
        public int WindowID { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public List<string> ImageURLs { get; set; }
        public int Direction { get; set; }
        public int StepIncrement { get; set; }
        public string HoldForTicks { get; set; }
        public int CurrentImageIndex { get; set; }
        public int Slide { get; set; }
        public int HoldCountDown { get; set; }
        public object Tag { get; set; }
        public int TabStopIndex { get; set; }

        public CCLImageSliderProps()
        {
            ImageURLs = new List<string>();
        }
    }
}